assemblies/nexus-base-template/src/main/resources/overlay/etc/nexus.properties

```diff
@@ -4,7 +4,7 @@ application-host=0.0.0.0
 nexus-context-path=/
 
 # Nexus section
-nexus-work=${nexus-base}/../sonatype-work/nexus
+nexus-work=${karaf.data}
```

Comment:
`${karaf.data}/nexus` might be clearer and avoid potential clashes with karaf directories going forwards